src/markdown/how-it-works/onshore-oil-gas.md

```diff
@@ -85,8 +85,6 @@ permalink: /how-it-works/onshore-oil-gas/
           <process-step stepName="Learn more">
             <ul class="list-bullet">
             <li><a href="https://www.blm.gov/programs/energy-and-minerals/oil-and-gas/leasing">Leasing of Onshore Oil and Gas Resources</a></li>
-            <li><a href="https://www.blm.gov/or/news/files/BLM-SoundInvestmentforAmerica.pdf">The BLM: A Sound Investment for America (PDF)</a></li>
-            </ul>  
```

Comment:
@Maroyafaied This won't break anything, but you want to keep that `</ul>` at the bottom. You want to remove everything between `<li>` and `</li>`, but you want to keep that closing unordered list tag, since you kept a list item (`<li`>) there.

It broke the build before because it wasn't in markdown (it was in JSX), but we can chat about that some other time.